docs/section5/Rule5-19.md

```diff
@@ -18,34 +18,32 @@
 
-    - Get matching building segment in P_RMR: `building_segment_p = match_data_element(P_RMR, BuildingSegments, building_segment_b.id)`
-
-      - Calculate building segment window wall ratio in P_RMR: `building_segment_wwr_p = window_wall_areas_dictionary_p[building_segment_p][0] / window_wall_areas_dictionary_p[building_segment_p][1]`
+    - Calculate building segment window wall ratio in P_RMR: `area_type_wwr_p = window_wall_areas_dictionary_p["NONE"]["TOTAL_WINDOW_AREA"] / window_wall_areas_dictionary_p["NONE"]["TOTAL_WALL_AREA"]`
 
       **Rule Assertion:**
 
-      - Case 1: If building is all new and building segment window-wall-ratio is equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( building_segment_wwr_b == min(building_segment_wwr_p, 0.4) ): PASS`
+      - Case 1: If building is all new and window-wall-ratio for area type not in Table G3.1.1-1 is equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( area_type_wwr_b == min(area_type_wwr_p, 0.4) ): PASS`
 
-      - Case 2: Else if building is all new and building segment window-wall-ratio is not equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( building_segment_wwr_b != min(building_segment_wwr_p, 0.4) ): FAIL`
+      - Case 2: Else if building is all new and window-wall-ratio for area type not in Table G3.1.1-1 is not equal to that in P_RMR or 40%, whichever is smaller: `if ( B_RMR.building.is_all_new ) AND ( area_type_wwr_p != min(area_type_wwr_p, 0.4) ): FAIL`
 
-      - Case 3: Else if building is not all new and building segment window-wall-ratio is equal to that in P_RMR or 40%, whichever is smaller: `if ( NOT B_RMR.building.is_all_new ) AND ( building_segment_wwr_b == min(building_segment_wwr_p, 0.4) ): CAUTION and raise_warning "BUILDING IS NOT ALL NEW AND BASELINE WWR MATCHES THE SMALLER OF PROPOSED DESIGN WWR OR 40%. HOWEVER, THIS RULE DOES NOT APPLY TO THE EXISTING ENVELOPE PER TABLE G3.1 BASELINE COLUMN #5 (C). FOR EXISTING ENVELOPE, THE BASELINE FENESTRATION AREA MUST EQUAL THE EXISTING FENESTRATION AREA PRIOR TO THE PROPOSED WORK. A MANUAL CHECK IS REQUIRED TO VERIFY COMPLIANCE."`
+      - Case 3: Else if building is not all new and window-wall-ratio for area type not in Table G3.1.1-1 is equal to that in P_RMR or 40%, whichever is smaller: `if ( NOT B_RMR.building.is_all_new ) AND ( area_type_wwr_b == min(area_type_wwr_p, 0.4) ): CAUTION and raise_warning "BUILDING IS NOT ALL NEW AND BASELINE WWR MATCHES THE SMALLER OF PROPOSED DESIGN WWR OR 40%. HOWEVER, THIS RULE DOES NOT APPLY TO THE EXISTING ENVELOPE PER TABLE G3.1 BASELINE COLUMN #5 (C). FOR EXISTING ENVELOPE, THE BASELINE FENESTRATION AREA MUST EQUAL THE EXISTING FENESTRATION AREA PRIOR TO THE PROPOSED WORK. A MANUAL CHECK IS REQUIRED TO VERIFY COMPLIANCE."`
 
```

Comment:
We decided to change CAUTION to UNDETERMINED. But this could also be updated at the implementation stage.